### PR TITLE
RavenDB-13840 attempted to delete value but it wasn't in index

### PR DESCRIPTION
**Pull request by karmeli87**
seems to fix the repro.
this is draft to let the test suit run.

### CODE REVIEW

**Comment by karmeli87** [comment]
I think that regardless to this fix we have AVE in 32 bit :/

File: src/Raven.Server/Config/Categories/StorageConfiguration.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ public class StorageConfiguration : ConfigurationCategory
         public PathSetting TempPath { get; set; }
 
         [Description("Use the 32 bits memory mapped pager, even when running in 64 bits")]
-        [DefaultValue(false)]
+        [DefaultValue(true)]
```
Comment (ayende) [inline src/Raven.Server/Config/Categories/StorageConfiguration.cs]: Need to revert this

**Review by ayende — COMMENTED** [review]
The issue is that we are deleting the active section and then allocating from it?

File: src/Raven.Server/Config/Categories/StorageConfiguration.cs (hunk 1 above)
Comment (karmeli87) [reply src/Raven.Server/Config/Categories/StorageConfiguration.cs]: This is draft and I want it to run on 32 bits (64 is by default)

**Comment by karmeli87** [comment]
I'm still not 100% sure, but... 
The way we are getting the `ActiveDataSmallSection` is as following:
https://github.com/ravendb/ravendb/blob/71ea31731fbd3ead4aa0d09ef2902d4ba0d5340c/src/Voron/Data/Tables/Table.cs#L44

And when we delete a section we forget to update this value, so in the next tx we loading the wrong page.

**Review by ayende — APPROVED** [review]

**Comment by karmeli87** [comment]
Closing and reopen in 4.1 since is has to be done there also. (https://github.com/ravendb/ravendb/pull/9520)

**Pull request closed without merging**